packages/calypso-e2e/src/lib/pages/external/github-login-page.ts

```diff
@@ -0,0 +1,99 @@
+import { Page } from 'playwright';
+
+const selectors = {
+	// General
+	buttonContainingText: ( text: string ) => `button:has-text("${ text }")`,
+	buttonWithExactText: ( text: string ) => `button:text-is("${ text }")`,
```

Comment:
Instead of these selectors we can use `page.getByRole`, right?

Reply:
I'll replace it with different approach. Thanks for the feedback.